test/testData/sampleAnalyticsTemplateData.ts

```diff
@@ -0,0 +1,240 @@
+export const variables = [
+  '{',
```

Comment:
@CristiCanizales why not use a plain object and then do a JSON.stringify?

Reply:
I am following the exact same pattern we're using for the rest of the steps, for consistency